src/watts/plugin.py

```diff
@@ -72,7 +72,7 @@ def postrun(self, params, name) -> Results:
     def plugin_name(self):
         return type(self).__name__[6:]
 
-    def __call__(self, params: Parameters = None, name: str = 'Workflow', **kwargs) -> Results:
+    def __call__(self, params: Parameters = None, name: str = '', verbose=True, **kwargs) -> Results:
```

Comment:
I am not sure I see the value in the `verbose` option if the only use is the hide the `[watts] Calling PyARC ...` printout - I would recommend simplifying the code and removing the `verbose` option.

Reply:
This is actually a recognized best practice for libraries to give users control over what output is shown. As one example, I'll mention a snippet of the [E4S community policies](https://e4s-project.github.io/policies.html):
> *User-managed I/O Control* No package should have hardwired print or I/O statements that cannot be turned off through a programmatic interface; output should never be hard-wired to stdout or stderr. It is recommended that packages provide a way for users to turn on output and allow them to direct where it goes. Also, packages may print to stdout by default but only on one process (i.e., root rank “0”). But packages may also be completely silent by default (and require that users turn on outputting in the appropriate way).

Right now, we are forcing our output on users without given them an option to use watts in a "quiet" mode.